test/ssc_test/cmod_battwatts_test.cpp

```diff
@@ -100,7 +100,7 @@ TEST_F(CMBattwatts_cmod_battwatts, ResidentialDefaults) {
     ssc_number_t peakKwDischarge = *std::max_element(batt_power_data.begin(), batt_power_data.end());
     ssc_number_t peakKwCharge = *std::min_element(batt_power_data.begin(), batt_power_data.end());
 
-    EXPECT_NEAR(peakKwDischarge, 1.98, 0.1);
+    EXPECT_NEAR(peakKwDischarge, 2.16, 0.1);
```

Comment:
I'm sure I missed it but is this change due to the DischargetoGrid defaulting to true?

Reply:
Yes - since the discharge is no longer limited to load, the peak discharge can increase.